sdcm/nemesis.py

```diff
@@ -5246,6 +5246,35 @@ def disrupt_grow_shrink_zero_nodes(self):
         znode = random.choice([node for node in self.cluster.zero_nodes if node.dc_idx == self.target_node.dc_idx])
         self.decommission_nodes(nodes=[znode])
 
+    def disrupt_serial_restart_elected_topology_coordinator(self):
+        """ Rolling restart of topology coordinator node,
+        should trigger new coordinator node election
+        """
+        if not self.target_node.raft.is_consistent_topology_changes_enabled:
+            raise UnsupportedNemesis("Consistent topology changes feature is disabled")
+
+        self.use_nemesis_seed()
+        num_of_restarts = random.randint(1, len(self.cluster.nodes))
+        election_wait_timeout = random.choice([1, 5, 10, 15])
```

Comment:
let's log debug these params, so it's clear how many restarts and timeout was selected.
In later message about random wait may be shortened a bit then.

Reply:
done